src/library_fs.js

```diff
@@ -292,21 +292,12 @@ FS.staticInit();` +
     // permissions
     //
     flagModes: {
-      '"r"': {{{ cDefine('O_RDONLY') }}},
```

Comment:
how did this work before with the extra quotes? Seems like something must have added them, but I'm not sure where or how?

Reply:
Hmm reverted... Its the strangest thing.. I guess its because it gets parsed by node and then re-serialized.  

If you write:

`flags = {"x" : 1 }`

Then you print `flags` you get this: `{ x: 1 }`.. and because our library code basically gets printed by the node we end up errasing the quotes.. so to keep the quotes we need to double quote.. very strange.

Also, dropping the quotes does't normally matter but I bet it effect closure..

Reply:
Reverted this part.